syslog.go

```diff
@@ -26,11 +28,13 @@ func SyslogLevelWriter(w SyslogWriter) LevelWriter {
 }
 
 func (sw syslogWriter) Write(p []byte) (n int, err error) {
+	p = DecodeIfBinaryToBytes(p, true)
```

Comment:
I'm not sure it is relevant to do that. What is the point to enable binary logging if the writer silently converts it back to JSON?

Reply:
This and console are writing to text destinations (syslog/console), outputing binary is not good - so i thought decoding is needed. in case of writing to these text destinations, we cannot delay the decoding of the binary to json. when writing to file or stream etc we can run a decoder.

An alternative for this is users run another process that takes these logs and decodes it and prints to console/syslog...

Reply:
I get it for console. For syslog we should not implement it in binary mode as it does not make sense.

Reply:
excluded syslog and hlog from binary support.